parm/config/config.fv3

```diff
@@ -51,6 +51,7 @@ case $case_in in
         export WRITE_GROUP_GFS=1
         export WRTTASK_PER_GROUP_GFS=$npe_node_max
         export WRTIOBUF="1M"
+	export OCNRES=100
```

Comment:
This is not an appropriate place to set `OCNRES`.
For now, please create a case block in `config.fcst`
Besides, this is only needed if `FRACGRID` is `YES` (I think).

Reply:
Can you also please pay attention to indentation.